test/wrapper.test.ts

```diff
@@ -342,15 +239,19 @@ describe('Tests Native Wrapper', () => {
         // @ts-ignore it is an error but it does not know the type.
         expect(error.message).toMatch('Native is disabled');
       }
-      expect(RNSentry.getStringBytesLength).not.toBeCalled();
       expect(RNSentry.captureEnvelope).not.toBeCalled();
     });
+    test('Encloses message to an object on Android', async () => {
+      // TODO:
+    });
+    test('Do not introduce empty breadcrumbs if undefined before on Android', async () => {
+      // TODO:
+    });
     test('Clears breadcrumbs on Android if mechanism.handled is true', async () => {
       NATIVE.platform = 'android';
 
       const event: Event = {
         event_id: 'event0',
-        message: 'test',
```

Comment:
Any reason to remove the `message`?

Reply:
Yes, I want to separate test of the breadcrumbs behaviour and the message replacement (that's why I had the empty test prepared.). So the test are actually testing what they say they do.